hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -3225,47 +3221,25 @@ public void testNetworkConfigUnambiguous_whenAdvancedNetworkDisabled() {
         assertEquals(9999, config.getNetworkConfig().getPort());
     }
 
+    @Override
+    @Ignore("XSD validation allows multiple config, no programmatic validation is implemented")
```

Comment:
This needs a little effort since there is always a `MEMBER` config. We should collect the already defined tags inside an `advanced-network` and throw if one is a duplicate. Not sure this is needed though.